pkg/polaris/graphql/azure/cloud.go

```diff
@@ -353,10 +361,15 @@ func (a API) UpgradeCloudAccountPermissionsWithoutOAuth(ctx context.Context, id
 	a.log.Print(log.Trace)
 
 	query := upgradeAzureCloudAccountPermissionsWithoutOauthQuery
+	var queryFeature any = feature.Name
```

Comment:
Is the name ever expected to not be a string? should we type this to string instead?

Reply:
Name is always a string. However, if the feature has permission groups (`len(feature.PermissionGroups) > 0`), we have to use a different query. The other query expects a feature and not a string, using `any` here means both values can be assigned to same variable.